src/aioice/ice.py

```diff
@@ -854,9 +854,12 @@ async def get_component_candidates(
         for address in addresses:
             # create transport
             try:
-                _, protocol = await loop.create_datagram_endpoint(
+                transport, protocol = await loop.create_datagram_endpoint(
                     lambda: StunProtocol(self), local_addr=(address, 0)
                 )
+                transport._sock.setsockopt(  # type: ignore
```

Comment:
Why do we need this type: ignore?

Also, I'd rather have a constant defined somewhere (UDP_SOCKET_BUFFER_SIZE) rather than repeating the magical 262144 value.

Reply:
After adding `transport._sock.setsockopt`, the mypy static type checker will report this error:
```txt
error: "BaseTransport" has no attribute "_sock"
```

The `ignore` comment aims to ignore this error.

Reply:
Ah yeah you shouldn't be ignoring this error, you're using a private attribute!

This is likely to break at any time, or with alternative loop implementations. Use get_extra_info and check the return value is not None before calling setsockopt.

https://docs.python.org/3/library/asyncio-protocol.html#asyncio.BaseTransport.get_extra_info

Reply:
I didn't notice there is this get_extra_info method. Thanks for your guidance.

Reply:
You seem to have both the old and the new code

Reply:
Sorry. Have removed the old code.